pkg/kubelet/kubelet.go

```diff
@@ -2617,33 +2629,41 @@ func (kl *Kubelet) setNodeStatus(node *api.Node) error {
 		}
 	}
 
-	updated := false
+	needUpdated := true
```

Comment:
I don't think the variable name is clear. Why don't we change to to `readyConditionUpdated := false`. I think the original default (false) is also more intuitive.